tests/test_objective_funs.py

```diff
@@ -2368,8 +2369,8 @@ class TestObjectiveNaNGrad:
         GenericObjective,
         LinearObjectiveFromUser,
         ObjectiveFromUser,
-        # TODO: add Omnigenity objective (see GH issue #943)
         Omnigenity,
+        EffectiveRipple,
```

Comment:
can move this up a bit to be in alphabetical order (we are testing it, just not using the default test). It's only the ones from user that we actually don't test at all.